Kickstarter-iOS/Views/Cells/BackingCell.swift

```diff
@@ -21,11 +37,32 @@ internal final class BackingCell: UITableViewCell, ValueCell {
 
     _ = self
       |> baseTableViewCellStyle()
-      |> BackingCell.lens.contentView.layoutMargins %~~ { layoutMargins, cell in
+      |> BackingCell.lens.contentView.layoutMargins %~~ { _, cell in
         cell.traitCollection.isRegularRegular
           ? .init(topBottom: Styles.grid(6), leftRight: Styles.grid(16))
-          : layoutMargins
+          : .init(topBottom: Styles.grid(3), leftRight: Styles.grid(2))
     }
+
+    _ = self.backingInfoButton
+      |> borderButtonStyle
+      |> UIButton.lens.title(forState: .normal) %~ { _ in Strings.backing_info_info_button() }
+      |> UIButton.lens.contentEdgeInsets .~ .init(topBottom: Styles.gridHalf(3),
+                                                  leftRight: Styles.gridHalf(5))
```

Comment:
oh, I did that because the button was really tall in that little space. I basically set it to match some of the other compressed height versions of this button we have, such as dashboard "view", friend follow buttons, and apply pay button. I will update to include bigger insets for ipad. Good catch, it means we need to update all the compressed buttons for ipad and this should probably be a base button style.